src/internal/connector/exchange/container_resolver_test.go

```diff
@@ -595,7 +595,7 @@ func (suite *FolderCacheIntegrationSuite) TestCreateContainerDestination() {
 
 	for _, test := range tests {
 		suite.T().Run(test.name, func(t *testing.T) {
-			folderID, err := CreateContainerDestinaion(
+			folderID, err := CreateContainerDestination(
```

Comment:
the phantom T strikes again!

Reply:
Hehe, I think that one has been hiding around for half a year, actually.